### PR TITLE
Add Dutch (and Belgian Dutch) matching labels

### PR DESCRIPTION
**Pull request by roelvangils**
This pull request adds Dutch (Netherlands) and Belgian Dutch language labels to your extension. These labels have been tested on some internal sample pages and work as expected. Thank you for considering this merge request! If you approve, we can also add French labels.

### CODE REVIEW

File: src/matching/autocomplete-dict.json [hunk 1]
```diff
@@ -56,150 +56,1096 @@
...
+            "upload"
+        ],
+        "44": [
+            "tel",
+            "telefoon",
+            "telefoonnummer",
+            "mobiel",
+            "gsm",
+            "werktelefoon",
+            "thuisnummer",
```
Comment (PhilippRecke) [inline src/matching/autocomplete-dict.json]: Please remove trailing comma. Causes JSON parsing error.

**Comment by PhilippRecke** [comment]
Hi, thank you very much for your contribution, which is greatly appreciated! I will go over the PR and add some comments.
The plugin needs additional logic in "matchingclasses.ts", as supported languages are currently hardcoded into the matching algorithm. I will add this in the coming days.
Feel free to create another PR for French, I'll be happy to add it as well. The more languages supported, the better. :)

File: src/matching/autocomplete-dict.json [hunk 2]
```diff
@@ -56,150 +56,1096 @@
...
+            "neuanmelden",
+            "neu",
+            "registrieren",
+            "registrierung"
+        ]
+    },
+    "byFormType_NL": {
+        "login": [
+            "login",
+            "log in",
```
Comment (PhilippRecke) [inline src/matching/autocomplete-dict.json]: Just an FYI: the plugin sanitizes all dictionary strings, thus removing whitespaces beforehand. "login" and "log in" will result in the same matching-string. However, this does not affect the outcome at all.

File: src/matching/autocomplete-dict.json [hunk 3]
```diff
@@ -56,150 +56,1096 @@
...
+            "verjaardagdag",
+            "dag"
+        ],
+        "39": [
+            "verjaardagmaand",
+            "maand"
+        ],
+        "40": [
+            "geboortejaar",
+            "jaar"
```
Comment (PhilippRecke) [inline src/matching/autocomplete-dict.json]: Possibly too generic. Might cause false positive matches, if the input is actually of type "cc-exp-year". I personally would opt to remove "jaar" from this list.

**Review by PhilippRecke — CHANGES_REQUESTED** [review]

**Comment by roelvangils** [comment]
@PhilippRecke, thank you for reviewing the PR and providing feedback. It's been busy, so I haven't had a chance to address the comments or add the French strings yet. I hope to do it this week! 🙂

**Comment by roelvangils** [comment]
Hi @PhilippRecke, I apologize for the super long delay in addressing these minor issues! ☺️ I completely lost track of this task. I now have more time to add the French labels as well.

**Comment by PhilippRecke** [comment]
Hi, totally fine, I lost track of it too. Again, thank you very much! If you want to invest the time to add French, that would still be awesome! :)
Do you think it would make sense to add the ability to enable/disable the languages to match against individually in the settings?
Or is it for your usecase ideal to allways check for all languages at the same time?

**Comment by roelvangils** [comment]
> Do you think it would make sense to add the ability to enable/disable the languages to match against individually in the settings? Or is it for your usecase ideal to allways check for all languages at the same time?

That's a very good question. I don't believe it will ever be necessary to test different language labels on the same page. However, the label checks should ideally be activated based on the page language specified in the HTML element, right?

**Pull request merged**